tokio/src/runtime/builder.rs

```diff
@@ -375,6 +381,26 @@ impl Builder {
             blocking_pool,
         })
     }
+
+    /// Sets a custom timeout for a thread in the `BlockingPool`.
+    ///
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use tokio::runtime;
+    /// # use std::time::Duration;
+    ///
+    /// # pub fn main() {
+    /// let rt = runtime::Builder::new()
+    ///     .keep_alive(Duration::from_millis(100))
+    ///     .build();
+    /// # }
+    /// ```
+    pub fn keep_alive(&mut self, duration: Duration) -> &mut Self {
```

Comment:
I think it might be worth giving this a name that indicates more clearly that it's specific to the blocking pool? Maybe
```suggestion
    pub fn blocking_worker_keep_alive(&mut self, duration: Duration) -> &mut Self {
```
(although that's kinda wordy) or 
```suggestion
    pub fn blocking_keep_alive(&mut self, duration: Duration) -> &mut Self {
```